### DISCUSSION TITLE
Wuerstchen Pipeline

### DISCUSSION THREAD

**Discussion by Acephalia** [seruva19/kubin#147] [Ideas]
@seruva19 hope you’ve been well. Thanks for all the updates and enhancements as always. 

I noticed that [wuerstchen](https://huggingface.co/docs/diffusers/main/en/api/pipelines/wuerstchen) is available as a diffusers pipeline now. Any chance we could try/use it with the Kubin GUI?

**Comment by seruva19** [comment]
The primary focus of Kubin from the beginning was to make it Kandinsky-only and create its own "ecosystem", so (at least currently), I don't have plans to integrate other models.
Also SD.Next is already doing well in providing universal frontend for various diffusers models, and I don't see any benefit in duplicating it.
Thank you for your understanding.

**Comment by Acephalia** [comment]
Completely fair and not a problem. Just thought I’d ask as I just like your interface better. I have tried sdnext with little to know luck getting Kandinsky or any other non SD model working.

**Comment by seruva19** [comment]
This might actually be a good idea to implement an extension that allows the use of other diffusers models. However, there are still many core functions I need to implement for the base application. For instance, I want to significantly enhance inpainting and outpainting, as they are quite limited at the moment. So, I may consider something like this in the future.

**Reply by Acephalia** [reply]
No pressure at all. Appreciate all that you are doing with this. 🖤

As a note for anyone else: I actually managed to try it using DiffusionMagic and really wasn’t that impressed. It’s fast as heck but really hit or miss and not consistent at all. The images on the project page are cherry picked to kingdom come I think.
